### PR TITLE
Add linter configs

### PR DESCRIPTION
**Pull request by markallenramirez**

### CODE REVIEW

File: angular.js [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+  },
+  plugins: [
+    'spellcheck',
+    'no-only-tests',
+  ],
+  rules: {
+    'spellcheck/spell-checker': (() => {
+      Array.prototype.push.apply(spellcheckRule[1].skipWords, [
+        'acc', // DXHotelsNetApi\Client\views\currentHotel.js ?
+        'accessor',
```
Comment (pomahtri) [inline angular.js]: this part is common for all frameworks, let's add this directly to `spell-checker.js`

File: angular.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
+/* eslint-env node */
```
Comment (williamvinogradov) [inline angular.js:1]: Let's remove the extended part and move all rules to the general rules section.
And we want to make this config extendable in other projects
So, let's remove all unnecessary parameters related to the project with dx examples.

And we should remove the spell-checker for this config.
We will extend it as a separate config for all file extensions.

```javascript
/* eslint-env node */
/* eslint-disable quote-props */

// eslint-disable-next-line import/no-extraneous-dependencies
const spellcheckRule = require('./spell-check').rules['spellcheck/spell-checker'];

module.exports = {
  plugins: [
    'no-only-tests',
  ],
  extends: [
    'eslint:recommended',
    'devextreme/typescript'
  ],
  rules: {
    'class-methods-use-this': 0, // TODO warn (was error)
    'func-names': 0, // TODO warn (was warn) >500
    'import/extensions': 0,
    'no-use-before-define': 0,
    'prefer-destructuring': 0,
    'no-param-reassign': ['error', { 'props': false }],
    'no-only-tests/no-only-tests': 'error',
    'eqeqeq': 0,
      'no-plusplus': 0,
      'max-len': 0,
      'max-classes-per-file': 0,
      'consistent-return': 0,
      'import/extensions': 0,
      'import/no-webpack-loader-syntax': 0,
      'no-restricted-properties': 0,
      'no-restricted-globals': 0,
      'spellcheck/spell-checker': 0,
      'no-mixed-operators': 0,
      'no-useless-concat': 0,
      'no-self-assign': 0,
      'default-case': 0,
      'no-var': 0,
      'no-return-assign': 0,
      'vars-on-top': 0,
      'no-sequences': 0,
      'new-cap': 0,
      'no-param-reassign': 0,
      'no-multi-assign': 0,
      'no-restricted-syntax': 0,
      'prefer-rest-params': 0,
      'radix': 0,
      'no-underscore-dangle': 0,
      'operator-assignment': 0,
      'prefer-const': 0,
      '@typescript-eslint/naming-convention': 0,
      '@typescript-eslint/no-throw-literal': 0,
      '@typescript-eslint/no-use-before-define': 0,
      '@typescript-eslint/no-shadow': 0,
      '@typescript-eslint/no-loop-func': 0,
      '@typescript-eslint/ban-types': 0,
      '@typescript-eslint/no-this-alias': 0,
      '@typescript-eslint/no-empty-function': 0,
      '@typescript-eslint/no-explicit-any': 0,
      '@typescript-eslint/no-unused-vars': 0,
      '@typescript-eslint/no-unused-expressions': 0,
      '@typescript-eslint/no-useless-constructor': 0,
      '@typescript-eslint/explicit-module-boundary-types': 0, // was warn
  },
};
```

File: jquery.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
+/* eslint-env node */
```
Comment (williamvinogradov) [inline jquery.js:1]: The same comment about the rules section and unnecessary project-related parameters
Take a look at ```angular.js```

File: react.js [hunk 4]
```diff
@@ -0,0 +1,207 @@
+/* eslint-env node */
```
Comment (williamvinogradov) [inline react.js:1]: The same comment about the rules section and unnecessary project-related parameters
Take a look at ```angular.js```

File: vue.js [hunk 5]
```diff
@@ -0,0 +1,174 @@
+/* eslint-env node */
```
Comment (williamvinogradov) [inline vue.js:1]: The same comment about the rules section and unnecessary project-related parameters
Take a look at ```angular.js```

**Review by williamvinogradov — APPROVED** [review]

**Pull request merged**